modules/dialogic_verbaliser/phrases.py

```diff
@@ -0,0 +1,105 @@
+from dialogic_util.random_tuple import RandomTuple
+
+
+class Phrases:
+    # get from config
+    connection_path = "../dialogic_resources/phrase_lists/segue-connecting-phrases.txt"
```

Comment:
Consider extracting "../dialogic_resources/phrase_lists/" to a variable/constant. Would be easier to check if all paths are the same :)